Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -204,17 +185,47 @@ function prepareMergeConcept(conceptId) {
         conceptsToMerge = conceptsToMerge + "," + conceptId;    
     }
 }
-
 function createWrapper(wrapperId) {
   window.location = '${pageContext.servletContext.contextPath}/auth/conceptlist/addconceptwrapper?wrapperId=' + wrapperId;
 }
-
+var conceptId;
+ 
+	$(document).ready(function(){
+	$(".btnReview").click(function() {
+		conceptId= $(this).closest("tr")   // Finds the closest row <tr> 
+	                       .find(".entryID")     // Gets a descendent with class="entryID"
```

Comment:
just add a data attribute and get the id from the attribute. it would be much cleaner and less error prone.

Reply:
actually, it already is in a data attribute, just get it from there